scripts/publish-gh-pages.sh

```diff
@@ -28,7 +28,7 @@ sed -i "s/IMAGE_TAG_OVERRIDE_WHEN_PUBLISHING/${NEW_TAG}/g" ./snyk-monitor/values
 sed -i "s/IMAGE_TAG_OVERRIDE_WHEN_PUBLISHING/${NEW_TAG}/g" ./snyk-monitor-deployment.yaml
 
 echo building new helm release
-helm init --client-only
+helm lint snyk-monitor
```

Comment:
Don't we still need the helm init command after it?

Reply:
Kinda seems like a v2 / Tiller thing, init doesn't exist in v3 Helm

Reply:
It's deprecated from helm v3

![image](https://user-images.githubusercontent.com/17513970/104168572-ea28a000-5406-11eb-87e0-418acd04dc90.png)

Reply:
another Q - what does `helm lint` does? I wonder if it is better to fail the build if you didn't manually lint the helm before you commit (like in the `test` script). otherwise we might publish a different version of the helm from what we have in staging/master